compliance/compliance.go

```diff
@@ -121,7 +141,7 @@ func (c *Compliance) createIndexMsg(report *v4.IndexReport, nodeName string) *se
 	}
 }
 
-func (c *Compliance) manageNodeScanLoop(ctx context.Context) <-chan *sensor.MsgFromCompliance {
+func (c *Compliance) manageNodeInventoryScanLoop(ctx context.Context) <-chan *sensor.MsgFromCompliance {
```

Comment:
it's unfortunate the two methods are almost identical, but since we have plans to remove the inventory stuff in the future, it's ok with me

Reply:
I separated them, because there were two switch-cases that were almost identical. In the approach with functions, I can at least name them, so that the readers know what is the difference.